### PR TITLE
added builds

### PR DESCRIPTION
**Pull request by marcstreeter**

### CODE REVIEW

**Comment by marcstreeter** [comment]
I'm really unsure on how to go about adding `projects`.  As you'd pointed out that the problem or rather difficulty with this was that `projects` has many subsections (`runs`, `tests`, `builds`).  So consider this my first try.  I'd appreciate any guidance you have here.

File: src/projects.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  }
+}
+
+/**
+ * Retrieve all builds of a project.
+ * @param {Object} user
+ * @param {String} projectName
+ * @param {Object} query
+ */
+Projects.prototype.builds = async function ({ user, projectName, query }) {
```
Comment (MiloCasagrande) [inline src/projects.js:23]: I probably would have use a more verbose approach like `findBuilds`, `findBuildById` etc... but this approach is fine as well.

Comment (marcstreeter) [reply src/projects.js:23]: lemme update this to conform to the rest of the project. I wasn't sure what to name anything .

Comment (marcstreeter) [reply src/projects.js]: fixed

**Pull request merged**